src/modules/users/users.controller.ts

```diff
@@ -85,4 +89,19 @@ export class UsersController {
       data: updatedUser,
     };
   }
+
+  @Put('/wishlist/:accommodationId')
+  @UseGuards(JwtAuthGuard)
+  @UpdateUserSwaggerDecorator()
+  async wishlistAccommodation(
+    @Param('accommodationId', new ParseUUIDV4Pipe()) accommodationId: string,
+    @Request() req: any,
```

Comment:
don't use any. we have GetUser decorator that you can use for getting user object from request.